demo/attester/attester.Dockerfile

```diff
@@ -58,7 +58,7 @@ ENV PATH="/root/.cargo/bin:/opt/rust/bin:${PATH}"
 # Install Parsec service
 RUN git clone -b attested-tls https://github.com/ionut-arm/parsec.git \
 	&& cd parsec \
-	&& git checkout dc561a5e4dee998ded29c7bd3310429341ddf237 \
+	&& git checkout 1ac2060531b391ff1f335369dc4d1e4f17aee1aa \
```

Comment:
Should we not use a fixed labels instead of commit hashes?

Reply:
What kind of labels?

I prefer commit hashes especially because they allow this sort of fixes to be applied. I guess git tags can be moved as well, but changing the commit means that Docker will pick up the change without having to rebuild the whole image from scratch.

Reply:
I meant checkout with a specific tag to be cleaner.

Reply:
Very little value in lots of granular tags at this point - this is coming from a prototype fork. These are not releases. Commit hashes fine for this purpose, IMO.